client/src/components/ListingItem.jsx

```diff
@@ -34,37 +35,39 @@ const ListingItem = ({ content, type }) => {
     }
   };
 
-  if (content.contentType === 'project') {
-    title = content.name;
-    person = 'Owner: ' + content.owner.name;
-    image = setIcon(content.featuredComponent.type, content);
-  } else if (content.contentType === 'component') {
+  if (content.contentType === 'component') {
```

Comment:
These if statements are kind of gross here is there a way we can refactor it in a way that parses the content object appropriately. Maybe that would look like making this a class based component. It will clean this code up significantly.